src/app/views/datasets/selectable-table/SelectableTableController.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    values: PropTypes.arrayOf(PropTypes.object)
+}
+
+export default class SelectableTableController extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            table: []
+        }
+
+        this.handleDetailsClick = this.handleDetailsClick.bind(this);
+    }
+
+    handleDetailsClick(event, key) {
+        if (event.target.tagName.toLowerCase() === "a") {
+            return;
+        }
+
+        event.preventDefault();
+        var isSelected = true;
+        var array = this.state.table;
+
+        this.state.table.map((item, index) => {
+            if (item === key) {
+                array.splice(index, 1);
+                this.setState({table: array});
+                isSelected = false;
+                return;
+            }
+        })
+
+        if (isSelected) {
+            array.push(key)
+            this.setState({table: array});
+        }
+    }
+
+    render() {
+        return (
+            <div className="selectable-table">
+                    <div className="selectable-table__heading">
+                        <ul className="selectable-table__list">
+                            <li className="selectable-table__list-item grid__col-6">Title</li>
```

Comment:
Because these share the class `selectable-table__list-item` they have a hover state which shows a pointer, which suggests it's clickable. Perhaps use a different classes for the heading list and items?